assets/themes/hailhydrant/hailhydrant.json

```diff
@@ -0,0 +1,601 @@
+{
+  "id": "HailHydrant",
+  "title": {
+    "en": "Hydrants, Extinguishers, Fire stations, and Rescue stations."
+  },
+  "shortDescription": {
+    "en": "Map to show hydrants, extinguishers, fire stations, and rescue stations."
+  },
+  "description": {
+    "en": "On this map you can find and update hydrants, fire stations,  rescue stations, and extinguishers in your favorite neighborhoods.  \n\nYou can track your precise location (mobile only) and select layers that are relevant for you in the bottom left corner. You can also use this tool to add or edit pins (points of interest) to the map and provide additional details  by answering  available questions.  \n\nAll changes you make will automatically be saved in the global database of OpenStreetMap and can be freely re-used by others."
+  },
+  "language": [
+    "en"
+  ],
+  "maintainer": "",
+  "icon": "https://wiki.openstreetmap.org/w/images/8/8c/Hailhydrant-logo.svg",
+  "version": "0",
+  "startLat": 13.67801,
+  "startLon": 121.6625,
+  "startZoom": 6,
+  "widenFactor": 0.05,
+  "socialImage": "",
+  "layers": [
+    {
+      "id": "hydrants",
+      "name": {
+        "en": "Map of hydrants"
+      },
+      "minzoom": 14,
+      "overpassTags": {
+        "and": [
+          "emergency=fire_hydrant"
+        ]
+      },
+      "title": {
+        "render": {
+          "en": "Hydrant"
+        }
+      },
+      "description": {
+        "en": "Map layer to show fire hydrants."
+      },
+      "tagRenderings": [
```

Comment:
To activate the images, add the string `"images"` as first entry in the `tagRenderings`.

That'll pick up images defined in the tags `image`, `image:0`, `image:1`, ..., `mapillary`, `wikidata`, `wikimedia`,...

There are a few more special values available, see https://github.com/pietervdvn/MapComplete/blob/master/assets/tagRenderings/questions.json